File: src/DonateTo.WebApplication/ClientApp/src/app/shared/models/organization.model.ts
import { BaseModel } from './base.model';
import { AddressModel } from './address.model';
import { ContactModel } from './contact.model';
import { UserOrganizationModel } from './user-organization.model';

export class OrganizationModel extends BaseModel {
  name: string;
  description: string;
  contact: ContactModel;
  addresses: AddressModel[];
  userOrganizations: UserOrganizationModel[];
}

File: src/DonateTo.ApplicationCore/Models/OrganizationModel.cs
using DonateTo.ApplicationCore.Entities;
using System.Collections.Generic;

namespace DonateTo.ApplicationCore.Models
{
    public class OrganizationModel
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public Contact Contact { get; set; }
        public IEnumerable<Address> Addresses { get; set; }
    }
}
